steward/src/cellars.rs

```diff
@@ -29,16 +29,22 @@ pub const ORACLE2: (U256, &str) = (
     U256([5, 0, 0, 0]),
     "c47278b65443ce71cf47e8455bb343f2db11b70e",
 );
+pub const ORACLE3: (U256, &str) = (
+    U256([5, 0, 0, 0]),
+    "26cde3f5db92ea91c84c838e664fe42dec1b6747",
+);
 pub const ALLOWED_PRICE_ORACLES: [(U256, &str); 2] = [ORACLE1, ORACLE2];
 pub const ALLOWED_CACHE_PRICE_ROUTER: [&str; 1] = [CELLAR_RYETH];
 
 // permissions
 
 pub const ALLOWED_V2_0_SETUP_ADAPTORS: [(&str, &str); 1] = [(CELLAR_RYUSD, ADAPTOR_CELLAR_V2)];
-pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 3] = [
+pub const ALLOWED_V2_2_CATALOGUE_ADAPTORS: [(&str, &str); 5] = [
     (CELLAR_RYETH, ADAPTOR_AAVE_V3_A_TOKEN_V1),
     (CELLAR_RYBTC, ADAPTOR_AAVE_V3_A_TOKEN_V1),
-    (CELLAR_RYBTC, ADAPTOR_CELLAR_V1),
+    (CELLAR_RYBTC, ADAPTOR_LEGACY_CELLAR_V1),
```

Comment:
Is this implying then that the currently deployed version, despite allowing RYBTC to add the legacy adaptor, didn't have the correct ABI to call?

Reply:
No, it's the correct ABI I just changed the name to clarify that it's the legacy cellar adaptor. Sanity check from seven seas would be nice though.

Reply:
I meant that the old allowance here was to let RYBTC use the legacy adaptor, but it's only as of this update that that adaptor has the ABI with a third argument.

Reply:
Oh yeah, unclear to me too. Steward didn't have definitions for it at all until now.